docs/update_bios.rst

```diff
@@ -0,0 +1,52 @@
+BIOS Updates
+============
+
+Below are the steps for updating the BIOS on the app and mon servers. You can also follow these steps to update the BIOS on your computer, such as the Admin Workstation Computer.
```

Comment:
It would be worth mentioning the hardware types that this is applicable to up front. These instructions are NUC-specific, and will not make sense for other hardware (including the Admin Workstation in most cases).

Reply:
Good point. I have updated the title and instructions to indicate this, as well as linked to our recommended hardware to explain why we're providing intel NUC instructions only.  see #5079f5e82

Reply:
I don't think this is clear enough yet. For example: the NUCs aren't mentioned until after the "prepare the USB stick" task, which may not apply to other hardware. Making the title more precise, and then presenting the steps for the F7 method as a numbered list, and then adding in provisos re: other methods or hardware, would probably help.